examples/browser-sdk/index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>AssemblyScript SDK Example</title>
+  </head>
+  <body>
+    <script src="/dist/as-bind.iife.js"></script>
+    <script src="./require.js"></script>
+    <script>
+      const SOURCE_CODE = `
+        export function test(): string {
+          return "ohai";
+        }`;
+
+      // A symbol allows us to store something as a global without risking
+      // name clashes or pollution.
+      const asbindTransform = Symbol();
+
+      // Shimming the require() call to *synchronously*
+      // return our transform module that we expect to be on a global.
+      // Any calls unrelated to the transform
+      // are implicitly passed through to RequireJS.
+      self.require = new Proxy(self.require, {
+        apply(target, thisArg, [path, ...args]) {
+          if (path !== "/dist/transform.js") {
+            return target.call(target, path, ...args);
+          }
+          return self[asbindTransform];
+        },
+        get(target, propName) {
+          if (propName !== "resolve") {
+            return target[propName];
+          }
+          return (path, ...args) => {
+            return path;
+          };
+        }
+      });
+
+      require(["/node_modules/assemblyscript/dist/sdk.js"], ({
+        asc,
+        assemblyscript
+      }) => {
+        // Register the `assemblyscript` property as a module of its own
+        // as is expected by most transform modules.
+        define("assemblyscript", assemblyscript);
+        // Load our ASBind transform...
+        require(["/dist/transform.amd.js"], transform => {
```

Comment:
Woooaaahhh that's interesting how you load a transform in the browser SDK :hushed: